Gulpfile.mjs

```diff
@@ -1,24 +1,26 @@
 "use strict";
 
-const plumber = require("gulp-plumber");
-const through = require("through2");
-const chalk = require("chalk");
-const newer = require("gulp-newer");
-const babel = require("gulp-babel");
-const fancyLog = require("fancy-log");
-const filter = require("gulp-filter");
-const gulp = require("gulp");
-const path = require("path");
-const rollup = require("rollup");
-const rollupBabel = require("@rollup/plugin-babel").default;
-const rollupBabelSource = require("./scripts/rollup-plugin-babel-source");
-const rollupCommonJs = require("@rollup/plugin-commonjs");
-const rollupJson = require("@rollup/plugin-json");
-const rollupNodePolyfills = require("rollup-plugin-node-polyfills");
-const rollupNodeResolve = require("@rollup/plugin-node-resolve").default;
-const rollupReplace = require("@rollup/plugin-replace");
-const { terser: rollupTerser } = require("rollup-plugin-terser");
-
+import plumber from "gulp-plumber";
+import through from "through2";
+import chalk from "chalk";
+import newer from "gulp-newer";
+import babel from "gulp-babel";
+import fancyLog from "fancy-log";
+import filter from "gulp-filter";
+import gulp from "gulp";
+import path from "path";
+import { rollup } from "rollup";
+import { fileURLToPath } from "url";
+import pluginBabel from "@rollup/plugin-babel";
+import rollupBabelSource from "./scripts/rollup-plugin-babel-source.js";
+import rollupCommonJs from "@rollup/plugin-commonjs";
+import rollupJson from "@rollup/plugin-json";
+import rollupNodePolyfills from "rollup-plugin-node-polyfills";
+import { default as rollupNodeResolve } from "@rollup/plugin-node-resolve";
```

Comment:
This is the same as
```js
import rollupNodeResolve from "@rollup/plugin-node-resolve";
```

However, I _think_ what you want is this:
```js
import _rollupNodeResolve from "@rollup/plugin-node-resolve";
const rollupNodeResolve = _rollupNodeResolve.default;
```

This is a bit unfortunate, but it's because the default export is always `module.exports` and not `module.exports.default` when importing CJS modules.